### PR TITLE
Voxels

### PR DESCRIPTION
**Pull request by andy645jh**
Change manifest app-id

### CODE REVIEW

File: org.flatpak.HelloVoxels.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "app-id": "org.flatpak.HelloVoxels",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
```
Comment (TingPing) [inline org.flatpak.HelloVoxels.json]: Please move to `18.08`.

File: org.flatpak.HelloVoxels.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+    "app-id": "org.flatpak.HelloVoxels",
```
Comment (TingPing) [inline org.flatpak.HelloVoxels.json]: Needs an app-id controlled by upstream.

**Comment by TingPing** [comment]
This isn't an actual app, just the template, forget to push something?

**Comment by barthalion** [comment]
Git isn't the perfect tool to host binary files. Maybe try creating separate git repo and creating a release so a tarball gets generated?

**Comment by andy645jh** [comment]
@TingPing yes, I forgot set some files up ,now  I already added the missing .xml files and updated the manifest file with the information about app and I took the advice from @barthalion and put the binary files into a server, now I have a question, should I do a new pull request? or is a push enough since this request is already open?

**Comment by bilelmoussaoui** [comment]
The app-id seems to be wrong. You don't own that domain name... Plus is there a reason for using the GNOMD runtime instead of the Freedesktop one?

**Pull request closed without merging**